COMPATIBILITY.md

```diff
@@ -0,0 +1,30 @@
+# Support and Compatibility for fabric-contract-api-go
+
+GitHub is used for code base management, issues should be reported in the [FABCAG](https://jira.hyperledger.org/projects/FABCAG/issues) component in JIRA.
+
+## Summary of Compatibility
+
+This table shows the summary of the compatibility of the package at version 2.0, together with the Go version it requires and the Fabric Peer version it can communicate with.
+
+|                       | Peer Connectivity v2.0 | Go 1.13 |
+| --------------------- | ---------------------- | ------- |
+| Packages **v2.0**     | Yes                    | Yes     |
```

Comment:
I'm more in favour of having a "Go" column, with the entry in the table being the value that is supported, what do you think?
i.e. https://github.com/hyperledger/fabric-gateway-java#compatibility